pyproject.toml

```diff
@@ -34,6 +34,7 @@ types-colorama = "^0"
 types-requests = "^2"
 protobuf = "^4"
 pathlib = "^1.0.1"   
+qdrant-client = "^1.6.4"
```

Comment:
Should we make it optional? Just in case users want to use other vectordb or do not want to use vectordb.